dev-env/windows/manifests/msys2.json

```diff
@@ -1,7 +1,7 @@
 {
     "homepage": "http://msys2.github.io",
     "version": "20220118",
-    "url": [
+    "_url": [
```

Comment:
Is this here as a backup or does it have a meaning?

Reply:
This is ignored by everything currently. I thought it would be useful to keep track of where the artifacts were originally found. This is not a great long-term solution as there's nothing keeping that in sync with the "local copies" in the GCS bucket, but I thought designing a good long-term solution could wait until we have a working CI again.

Reply:
Probably worth adding a more descriptive string that `_url`, just in case someone wonders about the meaning and you're not around.